sfdx-project.json/sfdx-project.schema.json

```diff
@@ -84,41 +106,79 @@
       "description": "By default, the OAuth port is 1717. However, change this port if this port is already in use, and you plan to create a connected app in your Dev Hub org to support JWT-based authorization."
     },
     "plugins": {
+      "title": "CLI Plugins custom settings",
       "type": "object",
-      "description": "Salesforce CLI plugins and configuration used with this project."
+      "description": "Salesforce CLI plugin configurations used with this project.",
+      "additionalProperties": true
     },
     "packageAliases": {
+      "title": "Aliases for packaging ids",
       "type": "object",
-      "description": "The Salesforce CLI updates this file with the aliases when you create a package or package version. You can also manually update this section for existing packages or package versions. You can use the alias instead of the cryptic package ID when running CLI force:package commands."
+      "description": "The Salesforce CLI updates this file with the aliases when you create a package or package version. You can also manually update this section for existing packages or package versions. You can use the alias instead of the cryptic package ID when running CLI force:package commands.",
+      "patternProperties": {
+        "^[ -~]+$": {
+          "type": "string",
+          "pattern": "^(0Ho|033|05i|04t)([a-zA-Z0-9]{15}|[a-zA-Z0-9]{12})$"
+        }
+      }
     }
   },
   "definitions": {
+    "features": {
+      "type": "array",
+      "title": "Features",
+      "description": "The list of feature names to enable",
+      "oneOf": [
```

Comment:
Based on the docs (https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev2gp_config_file.htm) the packaging configuration does not support a `features` array. Where is this coming from ?

Reply:
It came from toolbelt but this definition is never even referenced in the schema, so it wouldn't even show up. I'm guessing the definition usage was removed in toolbelt but the definition was not. This should just be removed.